sdk/eventhubs/azure_messaging_eventhubs/perf.yml

```diff
@@ -0,0 +1,35 @@
+parameters:
+- name: PackageVersions
+  displayName: PackageVersions (regex of package versions to run)
+  type: string
+  default: 'source'
+- name: Tests
+  displayName: Tests (regex of tests to run)
+  type: string
+  default: '.*'
+- name: Arguments
+  displayName: Arguments (regex of arguments to run)
+  type: string
+  default: '.*'
```

Comment:
More so feedback to @gearama, but package and test selection don't support regex. Package selection does support basic Unix glob patterns, but test selection is just a simple match:

1. https://doc.rust-lang.org/cargo/commands/cargo-test.html (package selection)
2. https://doc.rust-lang.org/rustc/tests/index.html#filters (test selection

Reply:
@mikeharder Are these parameters mentioning "regex" common across all languages, or is there any problem with changing the names for rust just to match what our test package/name selection actually supports (unix glob patterns for packages but simple substring match for tests)?

Reply:
The parameters are passed into the shared perf automation, which does the regex matching:

https://github.com/Azure/azure-sdk-tools/blob/71a6250c9feb93e1bc6497bc3986e9498d87700c/tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/Program.cs#L159-L171